config/config.go

```diff
@@ -0,0 +1,17 @@
+package config
+
+import (
+	"os"
+
+	"github.com/joho/godotenv"
+)
+
+var (
+	PORT string
+)
+
+func init() {
+	godotenv.Load(".env")
```

Comment:
Why do we need to do this? We are anyways loading PORT from the environment variable? 🤔

Reply:
now, I have only one environment... but we can have logic where we can load specific .env file (like dev.env, prod.env files) based on the environment we use....

Reply:
We set the environment variables in the corresponding environments we run and don't commit these values in the codebase.
For example, locally you could set environment variables using the `export` command before building or running your go project.
Like: export $PORT=3000

Reply:
Another reason behind using environment variable is that it is environment-specific and not dependent on code. So, if you need to change an environment variable you need not make code commit or deployment. If we load environment variables from local files then it negates the reason for using environment variables in the first place.

Reply:
Thanks for the clarification :)

Reply:
you can see the changes [here](https://github.com/ashwinspg/explore-golang/pull/2/commits/dccc411ee715866b249fe8eafe5b30e46addcc34)